WalletWasabi.Fluent/ViewModels/NavBar/NavBarViewModel.cs

```diff
@@ -33,7 +35,10 @@ public NavBarViewModel(UiContext uiContext)
 				 .Subscribe();
 
 		Wallets = wallets;
+	}
 
+	public void Activate()
```

Comment:
Why is this extracted to a method?

Reply:
@soosr  Setting `WalletPageViewModel.IsSelected = true` will trigger a navigation, but at the time when the `NavBarViewModel` constructor runs the UIContext isn't completely initialized (which happens in `MainViewModel.cs:57` right after the NavBar is built.

I don't like this but I couldn't find a way around it. Ideas are welcome =)